name.py

```diff
@@ -6,8 +6,7 @@ def print_names(names):
 
 # Press the green button in the gutter to run the script.
 if __name__ == '__main__':
-    print_names(["Hugo", "Jeffrey"])
+    print_names(['Tolga',"Hugo", "Jeffrey"])
```

Comment:
Inconsistent style: You added your name with singular quotations marks instead of double.

Reply:
Well I don't care about you then